src/axom/spin/ImplicitGrid.hpp

```diff
@@ -1068,7 +1068,14 @@ ImplicitGrid<NDIMS, ExecSpace, IndexType>::QueryObject::visitCandidates(
       bool found =
         getVisitResult(candidatePredicate, iword * bitsPerWord + currBit);
       currBit++;
-      currBit += axom::utilities::countr_zero(currWord >> currBit);
+      if(currBit < numBits)
+      {
+        currBit += axom::utilities::countr_zero(currWord >> currBit);
+      }
+      else
+      {
+        currBit += axom::utilities::countr_zero(0);
+      }
```

Comment:
I'm not sure if this change is safe -- could it create an infinite loop?

If it's just a no-op in the `else` (i.e. adding `0`, the following might be better:
```suggestion
      currBit += (currBit < numBits) ? axom::utilities::countr_zero(currWord >> currBit) : 0;
```